pkg/machine/ignition.go

```diff
@@ -613,7 +614,14 @@ func GetProxyVariables() map[string]string {
 	proxyOpts := make(map[string]string)
 	for _, variable := range config.ProxyEnv {
 		if value, ok := os.LookupEnv(variable); ok {
-			proxyOpts[variable] = value
+			if value == "" {
+				continue
+			}
+
+			// TODO: use constants for host.containers.internal
```

Comment:
Leftover TODO comment?

Reply:
Don't worry, this will soon be cleaned up in another pull request 🧹